ocaml/xapi/message_forwarding.ml

```diff
@@ -3597,6 +3597,15 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
            forward_vdi_op ~local_fn ~__context ~self
              (fun session_id rpc -> Client.VDI.disable_cbt rpc session_id self))
 
+    let hidden_cbt_setter ~__context ~self ~value =
```

Comment:
Just like for `VDI.get_nbd_info`, we do not need to do forwarding here, because we only manipulate the DB. So we can just call the function in `Local` - see `get_nbd_info` below.